AUTHORS

```diff
@@ -19,3 +19,4 @@ Jinho Bang <jinho.bang@samsung.com>
 Taeyun Kim <twhy.kim@gmail.com>
 Umar Faruk <umar97faruk@gmail.com>
 Wuseok Jang <rufia00@gmail.com>
+Gwanhyung Lee <gwanhyung787@gmail.com>
```

Comment:
Please keep your name in alphabetical order.
So, it's better to move your name above Hyung Heo.
```
...
Gwanhyung Lee <gwanhyung787@gmail.com>
Hyung Heo <sunemo83@gmail.com>
...
```